### PR TITLE
PMM-9677 change admin password

### PR DESCRIPTION
**Pull request by yurkovychv**
[nightly](https://pmm.cd.percona.com/job/pmm2-ui-tests-nightly-temp/114/allure/)
[staging](https://pmm.cd.percona.com/blue/organizations/jenkins/aws-staging-start-temp/activity/)

### CODE REVIEW

File: pmm/pmm2-ui-tests.groovy [hunk 1]
```diff
@@ -226,11 +230,11 @@ pipeline {
                         waitForContainer('pmm-agent_mysql_5_7', "Server hostname (bind-address):")
                         waitForContainer('pmm-agent_postgres', 'PostgreSQL init process complete; ready for start up.')
                         sh """
+                            docker exec pmm-server change-admin-password \${ADMIN_PASSWORD}
```
Comment (atymchuk) [inline pmm/pmm2-ui-tests.groovy:233]: Do we need to change it if the user hasn't changed the default `admin`? In that case maybe not...

Comment (yurkovychv) [reply pmm/pmm2-ui-tests.groovy:233]: We are changing it by default for UI tests job. It takes a job parameter which has default value

File: pmm/aws-staging-start.groovy [hunk 2]
```diff
@@ -289,6 +289,7 @@ pipeline {
                                             ${DOCKER_VERSION}
                                         sleep 10
                                         docker logs \${VM_NAME}-server
+                                        docker exec \${VM_NAME}-server change-admin-password \${ADMIN_PASSWORD}
```
Comment (puneet0191) [inline pmm/aws-staging-start.groovy:292]: change-admin-password utility is only available in the latest PMM version, but if I want to launch an older PMM-Version it won't work.

File: pmm/pmm2-upgrade-tests.groovy [hunk 3]
```diff
@@ -170,12 +177,47 @@ pipeline {
...
+            steps {
+                sh """
+                    docker exec pmm-server grafana-cli --homepath /usr/share/grafana --configOverrides cfg:default.paths.data=/srv/grafana admin reset-admin-password \${ADMIN_PASSWORD}
+                """
+                script {
+                    env.ADMIN_PASSWORD = ADMIN_PASSWORD
+                }
+            }
+        }
+        stage('Don\'t set admin password for 2.26') {
```
Comment (puneet0191) [inline pmm/pmm2-upgrade-tests.groovy:211]: Do we have a bug here?

**Review by puneet0191 — CHANGES_REQUESTED** [review]

File: pmm/aws-staging-start.groovy (hunk 2 above)
Comment (yurkovychv) [reply pmm/aws-staging-start.groovy]: fixed thanks

**Review by puneet0191 — APPROVED** [review]

**Pull request merged**